ca_tools/scripts/path_printer.py

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+import math
+
+import rospy
+from GTSys import GTSys
+from geometry_msgs.msg import Pose, PoseStamped
+from nav_msgs.msg import Path
+
+class pathPrinter():
+
+    def __init__(self):
+        self._path = Path()
+        self._path_pub = rospy.Publisher('path', Path, queue_size=10)
+        self._gt = GTSys()
+        self._rate = rospy.Rate(1) #1 Path msg per second
+        self._rate.sleep()
```

Comment:
what is this sleep for?

Reply:
It was initially there for trying something. Not useful anymore. Deleted it.